SU-SWS/ace-sdssgryphon
# SDSS-453: gradient percentage change

# READY FOR REVIEW

# Summary
- Changed percentage in banner gradient to 50%

# Review By (Date)
- soon, it would be nice for the staging release.

# Urgency
- higher

# Steps to Test

1. Add a banner with a headline.
2. Inspect the banner and see it has a stop point of 50% instead of 30%.

# Affected Projects or Products
- SDSS Subtheme

# Associated Issues and/or People
[- SDSS-453](https://stanfordits.atlassian.net/browse/SDSS-453)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
